src/cmn/team.py

```diff
@@ -236,27 +236,28 @@ def gen_teamsvecs(cls, datapath, output, cfg):
                 data = scipy.sparse.vstack(data, 'lil')#{'bsr', 'coo', 'csc', 'csr', 'dia', 'dok', 'lil'}, By default an appropriate sparse matrix format is returned!!
```

Comment:
Hi @thangk 
this file has a conflict. please get the latest code, apply the change and send the pr.